<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WhereClauseWidgetForm</class>
 <widget class="QWidget" name="WhereClauseWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Where Clause</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QGroupBox" name="m_criteriaGroupBox">
       <property name="title">
        <string>Criteria</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QTabWidget" name="m_criteriaTabWidget">
              <property name="tabPosition">
               <enum>QTabWidget::West</enum>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="m_attrTab">
               <attribute name="title">
                <string>Attribute</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="0" column="0">
                   <layout class="QGridLayout" name="gridLayout_17">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Restrict Value</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QComboBox" name="m_restrictValueComboBox">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="1" column="0">
                   <layout class="QGridLayout" name="gridLayout_19">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_2">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Operator</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QComboBox" name="m_OperatorComboBox"/>
                    </item>
                   </layout>
                  </item>
                  <item row="2" column="0">
                   <layout class="QGridLayout" name="gridLayout_12">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_3">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Value</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <layout class="QGridLayout" name="gridLayout_16">
                      <item row="0" column="0">
                       <layout class="QGridLayout" name="gridLayout_13">
                        <item row="0" column="0">
                         <widget class="QRadioButton" name="m_valuePropertyRadioButton">
                          <property name="text">
                           <string>Property</string>
                          </property>
                         </widget>
                        </item>
                        <item row="1" column="0">
                         <widget class="QRadioButton" name="m_valueValueRadioButton">
                          <property name="text">
                           <string>Value</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item row="0" column="1">
                       <layout class="QGridLayout" name="gridLayout_15">
                        <item row="1" column="0">
                         <widget class="QComboBox" name="m_valueValueComboBox">
                          <property name="enabled">
                           <bool>false</bool>
                          </property>
                          <property name="minimumSize">
                           <size>
                            <width>200</width>
                            <height>0</height>
                           </size>
                          </property>
                          <property name="editable">
                           <bool>true</bool>
                          </property>
                         </widget>
                        </item>
                        <item row="0" column="0">
                         <widget class="QComboBox" name="m_valuePropertyComboBox">
                          <property name="enabled">
                           <bool>false</bool>
                          </property>
                          <property name="sizePolicy">
                           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                           </sizepolicy>
                          </property>
                          <property name="minimumSize">
                           <size>
                            <width>200</width>
                            <height>0</height>
                           </size>
                          </property>
                          <property name="editable">
                           <bool>false</bool>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="m_spatialTab">
               <attribute name="title">
                <string>Spatial</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_9">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_8">
                  <item row="0" column="0">
                   <layout class="QGridLayout" name="gridLayout_23">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_6">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Geometry Attribute</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QComboBox" name="m_geomAttrComboBox">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="1" column="0">
                   <layout class="QGridLayout" name="gridLayout_20">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_4">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Operator</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QComboBox" name="m_SpatialOperatorComboBox"/>
                    </item>
                   </layout>
                  </item>
                  <item row="2" column="0">
                   <layout class="QGridLayout" name="gridLayout_21">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_5">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Layer</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QComboBox" name="m_layerComboBox"/>
                    </item>
                   </layout>
                  </item>
                  <item row="3" column="0">
                   <widget class="QCheckBox" name="m_selectedObjCheckBox">
                    <property name="text">
                     <string>Use Selected Objects</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_22">
              <item row="0" column="1">
               <widget class="QComboBox" name="m_connectorComboBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QCheckBox" name="m_connectorCheckBox">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Connector</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_14">
            <item row="0" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="m_addWhereClausePushButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>17</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="m_removeWhereClausePushButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="m_clearAllPushButton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>17</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_18">
        <item row="0" column="0">
         <widget class="QTableWidget" name="m_whereClauseTableWidget">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>29</number>
          </attribute>
          <column>
           <property name="text">
            <string>Property</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Operator</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Connector</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_restrictValueComboBox</tabstop>
  <tabstop>m_OperatorComboBox</tabstop>
  <tabstop>m_valuePropertyRadioButton</tabstop>
  <tabstop>m_valuePropertyComboBox</tabstop>
  <tabstop>m_valueValueRadioButton</tabstop>
  <tabstop>m_valueValueComboBox</tabstop>
  <tabstop>m_connectorCheckBox</tabstop>
  <tabstop>m_connectorComboBox</tabstop>
  <tabstop>m_addWhereClausePushButton</tabstop>
  <tabstop>m_removeWhereClausePushButton</tabstop>
  <tabstop>m_whereClauseTableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_connectorCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_connectorComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_valuePropertyRadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_valuePropertyComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_valuePropertyRadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_valueValueComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_valueValueRadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_valueValueComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_valueValueRadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_valuePropertyComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
